.github/workflows/spack_cpu_build.yaml

```
@@ -111,25 +112,35 @@ jobs:
             mirrors:
               local-buildcache: oci://${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
               spack: https://binaries.spack.io/develop
+          EOF
 
       - name: Configure GHCR mirror
         run: spack -e . mirror set --oci-username ${{ env.USERNAME }} --oci-password "${{ secrets.GITHUB_TOKEN }}" local-buildcache
-      
+
       - name: Trust keys
         run: spack -e . buildcache keys --install --trust
-      
+
       - name: Find external packages
         run: spack -e . external find --all
-      
+
       - name: Spack develop ReSolve
         run: spack -e . develop --path=$(pwd) resolve@develop
 
       - name: Concretize
         run: spack -e . concretize
-  
-      - name: Install
-        run: spack -e . install --no-check-signature
-      
+
+      - name: Install dependencies
+        run: spack -e . install --no-check-signature --only dependencies
+
+      - name: Install package
+        run: spack -e . install --keep-stage --no-check-signature --no-cache --fresh
```

```suggestion
        run: spack -e . install --keep-stage --no-cache --fresh
```

I think one of `--no-cache` and `--fresh` aren't necessary, but we definitely don't need to check signatures at this stage as there shouldn't be anything to download...